workflows/speciesabundance.nf

```diff
@@ -120,6 +120,48 @@ workflow SpAnce {
     )
 }
 
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    SELECT KRAKEN2 and BRACKEN DATABASES
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+def select_kraken_database(kraken2_db) {
+
+    if(kraken2_db) {
+        kraken_database = Channel.value(file(kraken2_db))
+        log.debug "Selecting kraken2 database ${kraken_database} from '--kraken2_db'."
+    }
+    else {
+        error("Unable to select a kraken2 database: '--kraken2_db' was not provided")
+    }
+
+    return kraken_database
+}
+
+def select_bracken_database(bracken_db) {
+
+    if(bracken_db) {
+
+        def directory = new File(bracken_db)
+        def files = directory.listFiles()
```

Comment:
I'm actually wondering if on Azure we will have to have `--bracken_db` point to the individual bracken files. Or, since there are 2 files I believe, have two `--braken_db` parameters? Or some other solution. Could you test this branch on Azure to see what happens?

Reply:
So far the best solution is to give the **full** path to the local directory:
`"--bracken_db": "/.nextflow/assets/phac-nml/speciesabundance/tests/data/brackendatabase"`

Reply:
Listing files does not work with URIs in Azure, so I have updated the workflow and database tests for Azure functionality: [ad7bd8d](https://github.com/phac-nml/speciesabundance/pull/11/commits/ad7bd8d2fe51182f4380d6e17cc416425679c24a)

Reply:
Thank you so much Kyla for spending the time to test this out in Azure 😄 . I really appreciate it.